examples/pytorch/gat/test_gat.py

```diff
@@ -1,130 +0,0 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
where do you UT model_geometric?

Reply:
What?

Reply:
where are unit tests for `model_geometric` module?

Reply:
You told me to delete this in this PR.

Reply:
![Screenshot 2023-02-23 at 6 04 12 PM](https://user-images.githubusercontent.com/134559/221074232-48a857c0-9b47-4512-bcaa-a8915e1ec48f.png)

Where exactly did I say to delete it?

Reply:
That comment is from graphsage, you told me to delete it about a month ago. I'll readd tests for gat model similar to graphsage.

Reply:
Your link points to a file in main branch, please post a direct link to the comment you are referencing.

Reply:
There are 120 comments on this PR and several teams calls I am not going to pull up the comment... I was linking the old GAT unit test that is a repeat of the main.py with no model specific tests.